public/index.html

```diff
@@ -191,57 +193,82 @@
 </head>
 
 <body>
+<nav class="navbar fixed-bottom navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="#">#findthemasks</a>
+
+  <div class="ml-auto">
+    <ul class="navbar-nav">
+      <li class="nav-item dropup">
+        <a class="nav-link dropdown-toggle" href="#" id="locales-dropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Language
+        </a>
+        <div class="dropdown-menu dropdown-menu-right" aria-labelledby="locales-dropdown">
+          <a class="dropdown-item" href="?locale=en">English</a>
+          <a class="dropdown-item" href="?locale=fr-FR">French (France)</a>
+        </div>
+      </li>
+      <li class="nav-item dropup">
+        <a class="nav-link dropdown-toggle" href="#" id="countries-dropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Country
+        </a>
+        <div class="dropdown-menu dropdown-menu-right" aria-labelledby="countries-dropdown">
+          <a class="dropdown-item" href="/fr/index.html">France</a>
```

Comment:
need to figure out how we maintain the locale when switching between countries / pages in general. Should we use local storage to write the user preference?